docs: add links to forecast-actual format repos (#290)
cmu-delphi/forecast-eval: app/assets/about.md

```
  
  

> ##### Forecast evaluation

Reviewer: suggestion: More descriptive section heading would be nice. The difference in format is small, but maybe?

```suggestion
##### Forecasts with actuals
```

If you are interested in getting the forecasts paired with the corresponding actual values (if you were e.g. testing different evaluation methods), that can be found at [https://www.stat.berkeley.edu/~ryantibs/data/](https://www.stat.berkeley.edu/~ryantibs/data/ "this is a temporary home") in 3 (zipped) zip files.

> These files are static, and the latest forecast date is

Reviewer: suggestion: Be a little more specific about generation, maybe

```suggestion
These files are static, generated using [the aggregation script](link TBD), and forecast and actual data available on June 12, 2023. The latest forecast date available for each target signal is
```

Waiting on Ryan to share his code.

* [cases](https://www.stat.berkeley.edu/~ryantibs/data/cases.zip): 2023-02-13
* [hospitalizations](https://www.stat.berkeley.edu/~ryantibs/data/hospitalizations.zip):
  * 1 week: 2023-06-05
  * 2 week: 2023-06-05
  * 3 week: 2023-06-05
  * 4 week: 2023-06-05

> * [deaths](https://www.stat.berkeley.edu/~ryantibs/data/deaths.zip): 2023-03-06

Reviewer: Added the files to AWPS too, just in case. Files are still being uploaded, but the link should work shortly.

```
  
  

##### Forecasts with actuals

> If you are interested in getting the forecasts paired with the corresponding actual values (if you were e.g. testing different evaluation methods), that can be found at [https://www.stat.berkeley.edu/~ryantibs/data/](https://www.stat.berkeley.edu/~ryantibs/data/ "this is a temporary home") in 3 (zipped) zip files.

Reviewer: issue: One last link to update.

```suggestion
If you are interested in getting the forecasts paired with the corresponding actual values (if you were e.g. testing different evaluation methods), that can be found in [the Amazon S3 bucket](https://forecast-eval.s3.us-east-2.amazonaws.com/) in 3 zip files.
```

> These files are static, generated using [the aggregation script](https://raw.githubusercontent.com/cmu-delphi/forecast-eval/main/app/assets/forecastsWithActuals.md), and forecast and actual data available on June 12, 2023. The latest forecast date available for each target signal is

Reviewer: issue: File extension.

```suggestion
These files are static, generated using [the aggregation script](https://raw.githubusercontent.com/cmu-delphi/forecast-eval/main/app/assets/forecastsWithActuals.R), and forecast and actual data available on June 12, 2023. The latest forecast date available for each target signal is
```
